database/src/main/resources/db/migration/V1.50.0__NOTIF-603_add_org_id.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE email_aggregation
+  ADD COLUMN org_id text;
+
+CREATE INDEX ix_email_aggregation_org_id ON public.email_aggregation (org_id);
+
+ALTER TABLE event
+  ADD COLUMN org_id text;
+
+CREATE INDEX ix_event_org_id ON public.event (org_id);
```

Comment:
https://issues.redhat.com/browse/NOTIF-650

Reply:
thanks. The index modifying process is something we could document somewhere so everyone can do that in case of fire :) or is that documented already and I don' know about it?

Reply:
It's not documented yet. I need to finish my developer blog post 😃